mkt/search/tests/test_views.py

```diff
@@ -1252,3 +1254,66 @@ def test_suggestions_with_multiple_icons(self):
 
         for size in (128, 64, 48, 32):
             eq_(parsed[0]['icons'][str(size)], self.app2.get_icon_url(size))
+
+
+@patch('mkt.versions.models.Version.is_privileged', False)
+class TestMultiSearchView(RestOAuth, ESTestCase):
+    fixtures = fixture('user_2519', 'webapp_337141')
+
+    def setUp(self):
+        super(TestMultiSearchView, self).setUp()
+        self.url = reverse('api-v2:multi-search-api')
+        self.webapp = Webapp.objects.get(pk=337141)
+        self.shared_category = 'books'
+        self.webapp.update(categories=['books', 'business'])
+        self.website = website_factory(
+            description='something',
+            categories=json.dumps(['books', 'sports'])
+        )
+        self._reindex()
+
+    def tearDown(self):
+        for w in Webapp.objects.all():
+            w.delete()
+        unindex_webapps(list(Webapp.with_deleted.values_list('id', flat=True)))
+        for w in Website.objects.all():
+            w.delete()
+        Website.get_indexer().unindexer(_all=True)
+        super(TestMultiSearchView, self).tearDown()
+
+    def _reindex(self):
+        self.reindex(Website, 'mkt_website')
+        self.reindex(Webapp, 'webapp')
+
+    def test_verbs(self):
+        self._allowed_verbs(self.url, ['get'])
+
+    def test_has_cors(self):
+        self.assertCORS(self.anon.get(self.url), 'get')
+
+    def test_meta(self):
+        res = self.anon.get(self.url)
+        eq_(res.status_code, 200)
+        eq_(set(res.json.keys()), set(['objects', 'meta']))
+        eq_(res.json['meta']['total_count'], 2)
+
+    @patch('mkt.search.utils.statsd.timer')
+    def test_statsd(self, _mock):
+        self.anon.get(self.url)
+        assert _mock.called
+
+    def test_search(self):
+        res = self.anon.get(self.url)
+        eq_(res.status_code, 200)
+        objs = res.json['objects']
+        eq_(len(objs), 2)
+        eq_(objs[0]['id'], self.webapp.pk)
+        eq_(objs[1]['id'], self.website.pk)
```

Comment:
Is the order here based on something specific? Can it change somehow in the future? If both `pk`s are something like `1` can we also check for a type or something? This seems like a loose test is all I'm saying.